src/modules/Blur/Module.js

```diff
@@ -6,39 +63,39 @@ module.exports = function Blur(options,UI){
     options.title = "Blur";
     options.description = "Blur an Image";
     options.blur = options.blur || 2
-    
+
```

Comment:
The whole module.js content are written twice somehow, please look into it, my suggestion is clone our master separately, delete your blur/module.js and replace it with ours, that way no unnecessary changes would be made to it